src/Tree/types.ts

```diff
@@ -275,4 +275,9 @@ export interface TreeProps {
    * {@link Tree.defaultProps.transitionDuration | Default value}
    */
   transitionDuration?: number;
+  
+  /**
+   * If node is interactive, drag and pan d3 events will only trigger on main tree element and not on nodes and links.
+  */
+  interactiveNode?: boolean;
```

Comment:
In order to keep boolean prop naming conventions fairly aligned with the `has.../is.../should...` convention, `hasInteractiveNodes` would fit better:

```suggestion
  hasInteractiveNodes?: boolean;
```